src/components/VMenu/mixins/menu-keyable.js

```diff
@@ -60,20 +60,9 @@ export default {
       } else if (e.keyCode === 13 && this.listIndex !== -1) {
         this.tiles[this.listIndex].click()
       }
-
-      if (this.listIndex === -1) this.setActiveListIndex()
```

Comment:
Can you tell me what this was for? It doesn't seem to do anything useful as far as I can tell.